FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -88,13 +91,20 @@ public class FhirToDataLakeOrchestratorJob : IJob
 
         public int NumberOfPatientsPerProcessingJob { get; set; } = JobConfigurationConstants.DefaultNumberOfPatientsPerProcessingJob;
 
+        public int LowBoundOfProcessingJobResourceCount { get; set; } = JobConfigurationConstants.LowBoundOfProcessingJobResourceCount;
+
+        public int HighBoundOfProcessingJobResourceCount { get; set; } = JobConfigurationConstants.HighBoundOfProcessingJobResourceCount;
+
         public async Task<string> ExecuteAsync(IProgress<string> progress, CancellationToken cancellationToken)
         {
             _diagnosticLogger.LogInformation("Start executing FhirToDataLake job.");
             _logger.LogInformation($"Start executing FhirToDataLake orchestrator job {_jobInfo.Id}.");
 
             try
             {
+                _jobStatus = await InitializeJobStatusAsync(cancellationToken);
+                _result = JsonConvert.DeserializeObject<FhirToDataLakeOrchestratorJobResult>(_jobStatus.StatisticResult);
```

Comment:
If **JobResult** is not be assigned in constructor, maybe not mark it as private variable?

Reply:
Only be used by this class and should be unaccusable to others.